packages/contact-manager-rest-api/src/api-functions.ts

```diff
@@ -0,0 +1,127 @@
+import { checkAuth, sendErrorResponse, ISingleEndpointOpts } from '@sphereon/ssi-express-support'
+import { Request, Response, Router } from 'express'
+import { IRequiredContext } from './types'
+import Debug from 'debug'
+import { AddContactArgs } from '@sphereon/ssi-sdk.contact-manager'
+
+const debug = Debug('sphereon:ssi-sdk:contact-manager-rest-api')
+
+export function partyReadEndpoints(router: Router, context: IRequiredContext, opts?: ISingleEndpointOpts) {
+  if (opts?.enabled === false) {
+    debug(`Endpoint is disabled`)
```

Comment:
This says nothing and the console.logs were deliberate

Reply:
changed to console.log